db/migrate/20211027140313_migrate_customer_names.rb

```diff
@@ -0,0 +1,28 @@
+class MigrateCustomerNames < ActiveRecord::Migration[6.1]
+  class ApplicationRecord < ActiveRecord::Base
+    self.abstract_class = true
+  end
```

Comment:
Does this make sense? Would it be better to exclude migrations from this convention?

Reply:
I had some weird errors here initially, where Zeitgeist was complaining that the parent class was invalid. I guess it had loaded the files already and determined that `Enterprise` inherited from `ApplicationRecord`? :man_shrugging: 

Anyway, I've updated it now.

Reply:
Great! Zeitwerk 2.5 was just released two weeks ago.